website/src/components/v1/Container.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+
+export default class Container extends React.Component {
```

Comment:
function components?

Reply:
I let a class here as it's basically a copy-paste port of the legacy v1 component. 
But will transform it anyway as it would be cleaner. I want to add this comp on our v1->v2 migration cli anyway so it's better to clean it up :)